runner/package.json

```diff
@@ -11,7 +11,7 @@
     "scss:build": "node compile-scss.js",
     "copy-forms": "node copy-form-json.js",
     "clean:build": "rm -rf dist",
-    "dev": "yarn build && yarn start:local",
+    "dev": "NODE_ENV=development nodemon --watch src --ext js,json,ts,html,scss --exec 'yarn build && node dist/digital-form-builder-adapter/runner/index.js'",
```

Comment:
Please ignore this. hot reloading was broken so I fixed it on my branch. I should not have pushed it.

Reply:
I removed it.